### PR TITLE
DX: fix function calls

### PR DESCRIPTION
**Pull request by kubawerlos**
Part of going to strict code.

### CODE REVIEW

File: src/StdinFileInfo.php [hunk 1]
```diff
@@ -23,6 +23,7 @@ final class StdinFileInfo extends \SplFileInfo
 {
     public function __construct()
     {
+        parent::__construct('');
```
Comment (Wirone) [inline src/StdinFileInfo.php:26]: Does it actually fix something or is it here just to satisfy PHPStan? I prefer to ignore errors if there's no real problem reported.

Comment (keradus) [inline src/StdinFileInfo.php:26]:
```suggestion
        parent::__construct(__FILE__);
```
?

File: tests/Console/Command/SelfUpdateCommandTest.php [hunk 2]
```diff
@@ -47,8 +47,8 @@ protected function setUp(): void
 
         file_put_contents($this->getToolPath(), 'Current PHP CS Fixer.');
 
-        file_put_contents("{$this->root->url()}/{$this->getNewMinorReleaseVersion()}.phar", 'New minor version of PHP CS Fixer.');
-        file_put_contents("{$this->root->url()}/{$this->getNewMajorReleaseVersion()}.phar", 'New major version of PHP CS Fixer.');
+        file_put_contents($this->root->url().'/'.$this->getNewMinorReleaseVersion().'.phar', 'New minor version of PHP CS Fixer.');
```
Comment (keradus) [inline tests/Console/Command/SelfUpdateCommandTest.php:50]: why the change? what's wrong with original code?

File: src/StdinFileInfo.php (hunk 1 above)
Comment (kubawerlos) [reply src/StdinFileInfo.php]: It is missing a call to the parent constructor, that would set path value.

Explicit over implicit - better call it to set it than not set it and cross-fingers nothing will break.

File: tests/Console/Command/SelfUpdateCommandTest.php (hunk 2 above)
Comment (kubawerlos) [reply tests/Console/Command/SelfUpdateCommandTest.php]: I am noob: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7136/commits/f24ea6ad59308c6d025bddb06ceb566f9ddb5482

**Review by Wirone — APPROVED** [review]

File: tests/Console/Command/SelfUpdateCommandTest.php (hunk 2 above)
Comment (keradus) [reply tests/Console/Command/SelfUpdateCommandTest.php]: why changing interpolation to concatenation?

Comment (Wirone) [reply tests/Console/Command/SelfUpdateCommandTest.php]: @keradus because you can't interpolate call to static method.
